docker-compose/compose.yaml

```
@@ -268,9 +339,17 @@ services:
       - "--entryPoints.traefik.address=:8888"
       - "--accesslog=true"
       - "--providers.docker.exposedByDefault=false"
+      - "--ping.entrypoint=web"
+      - "--ping=true"
     ports:
       - "8080:8080"
       - "8888:8888"
+    healthcheck:
+      test: ["CMD", "traefik", "healthcheck", "--ping"]
```

```suggestion
      test: ["CMD", "traefik", "healthcheck"]
```

https://doc.traefik.io/traefik/operations/cli/#healthcheck

doesn't really seems required as per docs

---

Docs dont mention that but its required
```
# traefik healthcheck
Error calling healthcheck: please enable `ping` to use health check
```